### PR TITLE
Aiida 2 compatibility

### PR DESCRIPTION
**Pull request by ltalirz**

### CODE REVIEW

File: tests/test_graphql/test_full/test_full.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 data:
-  aiidaVersion: 1.6.9
+  aiidaVersion: 2.0.4.post0
```
Comment (ltalirz) [inline tests/test_graphql/test_full/test_full.yml:2]:
```suggestion
  aiidaVersion: str
```

**Comment by ltalirz** [comment]
@sphuber tests pass now - if you want to start early, feel free to resolve the conflicts & squash+merge

otherwise happy to take care of it myself

**Comment by sphuber** [comment]
Seems the codecov step is failing randomly in the `ci` workflow, it's really annoying. Not sure why that is happening

**Comment by ltalirz** [comment]
Looks like we're not the only ones seeing this... https://github.com/codecov/codecov-action/issues/557

**Comment by sphuber** [comment]
yeah, seems rate limiting issue

**Review by sphuber — APPROVED** [review]
Thanks @ltalirz

**Comment by ltalirz** [comment]
I have an account and I'll add the token

**Comment by sphuber** [comment]
While you're at it, maybe also look into the codecov badge in the README. It currently points to a 404

**Comment by ltalirz** [comment]
Fixed!

The plugin cutter is using coveralls.io. Historically, both services have had some reliability issues

**Pull request merged**